### ISSUE TITLE
Facebook endpoint deprecated

### ISSUE THREAD

**Issue by MoxorTheOne** [ideashower/ShareKit#364]
SHKFacebook share item is now showing

ERROR: This endpoint has been deprecated. To temporarily reenable it, you may disable the "august_2012" platform migration. It will be disabled permanently on August 1, 2012.

I just updated to the latest version with same result.

**Comment by markolo** [comment]
I have the same problem. It looks like ShareKit is using some code is going to be deprecated. Any help?

**Comment by saikatgupta** [comment]
I am also having the same issue . Is that something to do with configuration?

**Comment by pradyumnad** [comment]
Me too having the same problem

**Comment by saikatgupta** [comment]
Don't take code from ShareKit website. They are now obsolete. Take from github.

**Comment by adeelk** [comment]
give the steps to solve this problem

**Comment by weberland81** [comment]
same problem with twitter integration, anny solutions ???

**Comment by adeelk** [comment]
please do it ,solve this prblem

**Comment by ignacioorona** [comment]
Yep, I need to stick to this version! Any suggestions how to solve this problem? Now it's not even working!

**Comment by dzeitman** [comment]
Does anyone know if this issue has be addressed or solve?
